installers/osie/main.go

```diff
@@ -111,30 +150,17 @@ func (i Installer) kernelParams(ctx context.Context, action, state string, j job
 	}
 
 	// Only provide the Hollow secrets for deprovisions
-	if j.HardwareState() == "deprovisioning" && conf.HollowClientId != "" && conf.HollowClientRequestSecret != "" {
-		s.Args("hollow_client_id=" + conf.HollowClientId)
-		s.Args("hollow_client_request_secret=" + conf.HollowClientRequestSecret)
+	if j.HardwareState() == "deprovisioning" && i.hollowParams != "" {
+		s.Args(i.hollowParams)
 	}
 
 	if isCustomOSIE(j) {
-		s.Args("packet_base_url=" + osieBaseURL(j))
+		s.Args("packet_base_url=${base-url}")
 	}
 
 	if j.CanWorkflow() {
-		buildWorkerParams()
-		if dockerRegistry != "" {
-			s.Args("docker_registry=" + dockerRegistry)
-		}
-		s.Args("grpc_authority=" + grpcAuthority)
-		s.Args("grpc_cert_url=" + grpcCertURL)
+		s.Args(i.tinkParams)
```

Comment:
in my tests this appears to add a single space if `i.tinkParams == ""`.

Reply:
tinkParams will never be empty since its initialized with `grpc_authority` and `packet_base_url`.

Reply:
if `dataModelVersion != "1"` then `i.tinkParams == ""`

Reply:
hmm good point, this is still guarded by `CanWorkflow` which doesn't make any sense when dataModelVersion != 1. :thinking: ideally CanWorkflow returns false if dataModelVersion != 1 always, let me try that out.

Reply:
I'm going to make sure we're in tink mode as a prereq for the CanWorkflow checks. Will also open up an issue so that we drop CanWorkflow all together as boots doesn't really support a mixed-mode operation we're either in cacher, tink, standalone and the hw has no real say in it.

Reply:
I believe this has been fixed now, can you make sure please?